.changeset/beige-peaches-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@meilisearch/autocomplete-client": patch
```

Comment:
Should be `minor` for this kind of change, not patch.

Reply:
https://github.com/meilisearch/meilisearch-js-plugins/pull/1288#issuecomment-1933871566

Reply:
Sorry, done!